### PR TITLE
docs: add docs and conventions for adding images and contacts

### PR DESCRIPTION
**Pull request by miriamseely**
I recently put together a [Blog Post Checklist](https://docs.google.com/document/d/1Ri6l9Ql-4FiwIyCqQMc0GIT62_y2SLiUyg9z_vzcrc4/edit?usp=sharing) to document some of the conventions for blog posts and the process for getting them published. At Jake and Steve's suggestion, I've added some things from that checklist to the README in an effort to clarify the process for those wishing to publish content on our website. I did not include everything in the blog post checklist here, just a few general notes and explanations. Let me know what you think!

Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,8 @@ import DefinitionListItem from '@components/page/DefinitionListItem.astro';
 
 ### images
 
+Images used in content should always have lowercase file names and should be uploaded to the folder created for your content. Cover images of blog posts should have an aspect ratio of 1200x360. In order to insert an image into the body of your content, please include these lines of code at the top:
```
Comment (steveoh) [inline README.md:134]:
```suggestion
All images should have lowercase file names with words separated by hyphons aka kebab case. e.g.: `gis-day.png`.

Images should be organized into their pillar folder. e.g.: `/src/images/pillar-documentation/*`.

Blog posts started with the issue template will create a folder for your images in `/src/images/pillar-blog/{blog-slug}/`.

Aliases for these paths are available in the [/tsconfig.json](tsconfig). e.g.: `@images/blog/{blog-slug}` is equivalent to `/src/images/pillar-blog/{blog-slug}/`.

To update a blog post card and or the social sharing image for any page, update the pages or posts `cover_image` property with an image having an aspect ratio of 1200x630 pixels. 

In order to insert an image into the body of your content, please include these lines of code at the top:
```

File: README.md [hunk 2]
```diff
@@ -148,6 +150,17 @@ import DefinitionListItem from '@components/page/DefinitionListItem.astro';
...
+You should place this line of code in the spot where you want the image to appear in the content:
+
+- astro
+
+```astro
+
+  <Image src={photo} alt="text describing the image to a non sighted person" />
+  ```
+
+Remember to delete the placeholder file for your image after you have uploaded your images to the content folder. If you are not including images, be sure to delete the placeholder file before the content is merged.
```
Comment (steveoh) [inline README.md:162]:
```suggestion
When editing blog posts created by the issue template, remember to delete the placeholder file for your images after you have uploaded your images to the content folder. If you are not including images, be sure to delete the placeholder file before the content is merged.
```

**Review by steveoh — COMMENTED** [review]
These are great additions. Since this is getting long I might add a table of content links at the top to navigate all of this content.

Give my suggestions a read and edit them as you see fit. Let me know when you are ready for another look.

Thanks!

**Comment by miriamseely** [comment]
> These are great additions. I tried to separate, describe, and to make the advice more generic for all content on the website. Not just blog posts. Staff edit other page types as well.
> 
> Give my suggestions a read and edit them as you see fit. Let me know when you are ready for another look.
> 
> Since this is getting long I might add a table of content links at the top to navigate all of this content before we publish it.
> 
> Thanks!

Thank you for your suggestions, I made a few minor changes here and there but I think this is in pretty good shape. I think a table of contents is a great idea!

**Review by steveoh — APPROVED** [review]

**Pull request merged**